tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php

```diff
@@ -26,6 +26,9 @@ public function setUp() {
 		$this->module = new DeletePostsByTaxonomyModule();
 	}
 
+	/**
+	 * Test deleting posts from a single taxonomy term default post type.
+	 */
 	public function test_deleting_posts_from_built_in_taxonomy_terms() {
```

Comment:
Avoid function naming inconsistency across the different test cases. Either use `test_deleting` or `test_that_posts_are_deleted_...`

https://github.com/sudar/bulk-delete/blob/a239562313b008b47c6681fc3ff4c34fb305ed02/tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php#L32

https://github.com/sudar/bulk-delete/blob/a239562313b008b47c6681fc3ff4c34fb305ed02/tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php#L113